src/client/eppo-client.ts

```diff
@@ -113,15 +122,23 @@ export default class EppoClient implements IEppoClient {
     subjectAttributes: Record<string, any> = {},
     assignmentHooks?: IAssignmentHooks | undefined,
   ): string | null {
-    return (
-      this.getAssignmentVariation(
-        subjectKey,
-        flagKey,
-        subjectAttributes,
-        assignmentHooks,
-        ValueType.StringType,
-      ).stringValue ?? null
-    );
+    try {
+      return (
+        this.getAssignmentVariation(
```

Comment:
Since all these functions do is immediately delegate to `getAssignmentVariation()`, I'm thinking it would be cleaner to just have the `try`-`catch` in that method and forgo it in these user-facing syntactic sugar wrappers.